### PR TITLE
support environment variables for test configuration

### PR DESCRIPTION
**Pull request by ansyral**
Signed-off-by: xuzho <xuzho@microsoft.com>

#200

### CODE REVIEW

File: extension/src/Models/testConfig.ts [hunk 1]
```diff
@@ -17,5 +17,6 @@ export type RunConfigItem = {
     workingDirectory: string;
     args: any[];
     vmargs: any[];
+    env: {[key: string]: string; };
```
Comment (yaohaizh) [inline extension/src/Models/testConfig.ts]: If the value is not required, should add ? for declaration

**Comment by thwfreak** [comment]
I think this new configuration should be added to vscode-java-test/extension/launch.test.schema.json

**Comment by ansyral** [comment]
@thwfreak nice catch! Would you like to open a pull request to help fix this?

**Comment by thwfreak** [comment]
@ansyral  done with #225 :wink:

**Pull request merged**